<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(52, 108, 212);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>540</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="searchPage">
     <widget class="QTextBrowser" name="resultsBrowser">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>150</y>
        <width>600</width>
        <height>300</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(52, 108, 212);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>100</y>
        <width>500</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(52, 108, 212);
background-color: rgb(255, 255, 255);
border-radius: 15px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelEnter">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>50</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(175, 204, 255);</string>
      </property>
      <property name="text">
       <string>Enter:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSearch">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>100</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-radius: 10px;
	color: rgb(52, 108, 212);
	background-color: rgb(175, 204, 255);
}
QPushButton:hover
{
	color: rgb(40, 82, 160);
	background-color: rgb(155, 192, 255);
}</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonCloseSearch">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-radius: 10px;
	color: rgb(52, 108, 212);
	background-color: rgb(175, 204, 255);
}
QPushButton:hover
{
	color: rgb(40, 82, 160);
	background-color: rgb(155, 192, 255);
}</string>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelName">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>50</y>
        <width>500</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(175, 204, 255);</string>
      </property>
      <property name="text">
       <string>KIMIO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonGoAdd">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-radius: 10px;
	color: rgb(52, 108, 212);
	background-color: rgb(175, 204, 255);
}
QPushButton:hover
{
	color: rgb(40, 82, 160);
	background-color: rgb(155, 192, 255);
}</string>
      </property>
      <property name="text">
       <string>Add Document</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="resultPage">
     <widget class="QTextEdit" name="resultsEdit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>700</width>
        <height>400</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(52, 108, 212);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonGoBack">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-radius: 10px;
	color: rgb(52, 108, 212);
	background-color: rgb(175, 204, 255);
}
QPushButton:hover
{
	color: rgb(40, 82, 160);
	background-color: rgb(155, 192, 255);
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonCloseResult">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-radius: 10px;
	color: rgb(52, 108, 212);
	background-color: rgb(175, 204, 255);
}
QPushButton:hover
{
	color: rgb(40, 82, 160);
	background-color: rgb(155, 192, 255);
}</string>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="addPage">
     <widget class="QPushButton" name="buttonCloseAdd">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-radius: 10px;
	color: rgb(52, 108, 212);
	background-color: rgb(175, 204, 255);
}
QPushButton:hover
{
	color: rgb(40, 82, 160);
	background-color: rgb(155, 192, 255);
}</string>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonGoSearch">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-radius: 10px;
	color: rgb(52, 108, 212);
	background-color: rgb(175, 204, 255);
}
QPushButton:hover
{
	color: rgb(40, 82, 160);
	background-color: rgb(155, 192, 255);
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="editPath">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>180</y>
        <width>500</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(52, 108, 212);
background-color: rgb(255, 255, 255);
border-radius: 15px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelPath">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>120</y>
        <width>311</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(175, 204, 255);</string>
      </property>
      <property name="text">
       <string>Enter the path of the document here:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelSuccess">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>270</y>
        <width>300</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(175, 204, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonPath">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>180</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
	border-radius: 10px;
	color: rgb(52, 108, 212);
	background-color: rgb(175, 204, 255);
}
QPushButton:hover
{
	color: rgb(40, 82, 160);
	background-color: rgb(155, 192, 255);
}</string>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
